docs/charon/charon-cli-reference.md

```diff
@@ -358,4 +358,120 @@ Flags:
       --p2p-relays strings                Comma-separated list of libp2p relay URLs or multiaddrs. (default [https://0.relay.obol.tech])
       --p2p-tcp-address strings           Comma-separated list of listening TCP addresses (ip and port) for libP2P traffic. Empty default doesn't bind to local port therefore only supports outgoing connections.
 ```
-You can also consider adding [alternative public relays](../faq/risks.md) to your cluster by specifying a list of `p2p-relays` in [`charon run`](#run-the-charon-middleware).
\ No newline at end of file
+You can also consider adding [alternative public relays](../faq/risks.md) to your cluster by specifying a list of `p2p-relays` in [`charon run`](#run-the-charon-middleware).
+
+## The `exit` subcommand
+
+The `exit` subcommands allows users to exit a validator currently existing in a Charon cluster.
+
+Considering the distributed nature of a Charon cluster, the `exit` commands enables users to submit their partially signed exit messages to a remote API, which will hold it for them until there are enough to be successfully aggregated in a fully signed exit message.
+
+Operators can then use the `exit` subcommand to download and broadcast the fully signed exit message to their Beacon Node.
+
+To facilitate custom deployments, the `exit` subcommands allows listing all the active validators contained in a cluster, as well as broadcasting pre-signed exit messages from a file.
+
+All of the commands require a valid lock file and `charon-enr-private-key` to authenticate requests to the remote API.
+
+```markdown
+Sign and broadcast distributed validator exit messages using a remote API.
+
+Usage:
+  charon exit [command]
+
+Available Commands:
```

Comment:
What is unclear for a typical user - what exact command(s) it needs to execute to achieve Exit?
It would be great to see a small paragraph describing the overall workflow for ordinary deployments.

Reply:
Done!